packages/shylu/shylu_node/fastilu/src/shylu_fastilu.hpp

```diff
@@ -115,19 +271,20 @@ class MemoryPrimeFunctorNnzCoo;
 template<class Ordinal, class Scalar, class ExecSpace>
 class MemoryPrimeFunctorNnzCsr;
 
-template<class Ordinal, class Scalar, class ExecSpace>
+template<class Ordinal, class Scalar, class ExecSpace, bool BlockCrsEnabled=false>
```

Comment:
Decided to use a template to enable the blockcrs capability. I didn't want to deal with performance issues related to runtime checks.